packages/cli/src/commands/test.js

```diff
@@ -41,6 +40,12 @@ export const builder = (yargs) => {
       type: 'boolean',
       default: true,
     })
+    .option('updateSnapshots', {
```

Comment:
Thinking out loud: for option name sometimes we use 'updateSnapshots' while others 'update-snapshots'. For CLI my preference is the latter.

Most of all I think we should be consistent.

Reply:
Never fear DP, yargs handles it all for us! 🏴‍☠️ Both are supported

Reply:
Updated to use dash case now. 🏴‍☠️ handles both for us, so no breaking change.